pom.xml

```diff
@@ -0,0 +1,5 @@
+<dependency>
```

Comment:
Try `libraryDependencies += ('com.github.tyagihas', 'scala_nats_2.11', '0.3.0')` in `build.sbt` instead of a `pom.xml` ;)